docs/blog/newsletter-june-2018.rst

```diff
@@ -0,0 +1,122 @@
+
+For an example of someone trying to explain the value of technical communication to folks who are new to it, `this LinkedIn post <https://www.linkedin.com/pulse/what-you-should-know-before-working-technical-writer-jerry-grohovsky>`_ highlights some interesting assumptions and pain points that might be worth thinking about.
+
+*If you're a lone docs person and still working on communicating this stuff to your team, pop into* `#lone-writer <https://writethedocs.slack.com/messages/lone-writer>`_ *to talk with folks who are/have been in your shoes.*
+
+-----------------------------------
+Junior vs. senior technical writers
+-----------------------------------
+A common theme in WTD career discussion is that the path into or through a documentation career is rarely straightforward. There are benefits to that, in terms of flexibility and a wide-ranging understanding of what documentarians actually *do*, day-to-day. But one of the drawbacks is that it can be difficult to see a clear path for advancement, even in more traditional 'tech comm' positions. This month, that topic was raised in the context of defining what constituted a **Senior Technical Writer** role versus a **Junior Technical Writer**?
+
+Our crowd of documentarians came up with three primary distinctions between the two (title aside, of course):
+* Amount and type of experience
+* Level of supervision required (or amount of supervision you're providing to others)
+* Degree and scope of non-writing responsibilities
+
+Senior writers would typically have more experience and require less supervision than junior writers. The senior writer may also be managing a team, having a junior writer produce docs, while the senior writer focuses on high-level planning, project management, and tooling decisions. However, some of these distinctions are only a helpful  for teams of multiple writers. For the lone writer, the key to advancing from junior to senior may lie in mastering certain skills (writing, tools, and technical competencies), collaborating cross-functionally on bigger projects, or developing specific expertise within the industry your company serves.
+
+*If you're struggling to define or move along your own career path, head on over to* `#career-advice <https://writethedocs.slack.com/messages/career-advice>`_ *where you'll find a wealth of constructive advice.*
+
+---------------------------------------
+Short advice for writing error messages
+---------------------------------------
+This month, the hivemind fielded a request for advice about writing good error messages. Attached to the request, though, was a with a familiar (and vital) caveat: They needed short, easy-to-digest pointers, rather than longer, meatier reads, in an effort to boost the odds that their developer teams would have the time to read what they shared.
+
```

Comment:
Remove "with a" in second line

Attached to the request, though, was a familiar...